packages/ledger-icp/src/mocks/ledger.mock.ts

```diff
@@ -2,7 +2,7 @@ import { Principal } from "@dfinity/principal";
 import { AccountIdentifier } from "../account_identifier";
 
 export const mockAccountIdentifier = AccountIdentifier.fromHex(
-  "3e8bbceef8b9338e56a1b561a127326e6614894ab9b0739df4cc3664d40a5958",
```

Comment:
This appears to be a hash of an ICP transaction: https://dashboard.internetcomputer.org/transaction/3e8bbceef8b9338e56a1b561a127326e6614894ab9b0739df4cc3664d40a5958

I replaced it with a valid account-id from another existing test.

Reply:
👉 https://github.com/dfinity/ic-js/pull/946